### ISSUE TITLE
My account has been permanently suspended

### ISSUE THREAD

**Issue by typedrat** [dimdenGD/OldTwitter#828]
I can't think of anything else that I could have done to violate the "X" Rules other than using this browser extension, and I have been permanently suspended from Twitter without any explanation. If I appeal, I immediately get a rejection email within thirty seconds.

I'd be careful using this.

**Comment by Stalbourne** [comment]
Yep, lost my account and seems like the creator got their account suspended too. https://x.com/dimdeneff

**Comment by kokoropaint** [comment]
Want to comment that my appeals have gone through but haven't been accepted yet, so I'm not sure. Someone else was able to get their account back. I wonder if too many bots are getting banned for similar reasons.

**Comment by infinitephantasm** [comment]
how'd you even manage to get in an appeal at all?? no matter how many i send, or how much i change the report text, i always just get a denial within only like 5 seconds

**Comment by TheUltraBadDuck** [comment]
I got the same problem. Did anyone solve this?

**Comment by wiichicken** [comment]
This happened to me too. Really fucking pissed at this.

**Comment by emmanu888** [comment]
I'm starting to think they're cracking down on extensions that changes the look of the site or how you use it.

And honestly i hate it.

**Comment by typedrat** [comment]
> how'd you even manage to get in an appeal at all?? no matter how many i send, or how much i change the report text, i always just get a denial within only like 5 seconds

That's exactly my experience, but that is still submitting an appeal.

**Comment by wiichicken** [comment]
> I'm starting to think they're cracking down on extensions that changes the look of the site or how you use it.
> 
> And honestly i hate it.

I don't think they're doing that specifically. OldTwitter is basically a new client, so I would somewhat understand why after the API changes.

**Comment by LabRatMime** [comment]
I just got suspended again, I'm awaiting my appeal.

**Comment by PanNyaa** [comment]
I also experienced the same issue (permanent account suspension) using the OldTwitterLayout. It happened a few days ago. I have never used OldTweetDeck, which allows you to use paid features for free, as I judged it to be too risky. Therefore, I cannot deny the possibility that users of OldTwitterLayout are being detected by Twitter staff in some way. Considering that this add-on has been removed from the Firefox add-on page (not once, but twice), it is also possible that "someone" has raised an objection to Mozilla.

**Comment by dimdenGD** [comment]
> I also experienced the same issue (permanent account suspension) while using the OldTwitterLayout a few days ago. I have never used OldTweetDeck, which allows you to use paid features for free, as I judged it to be too risky. Therefore, I cannot deny the possibility that users of OldTwitterLayout are being detected by Twitter staff in some way. Considering that this add-on has been removed from the Firefox add-on page not once, but twice, it is also possible that “someone” has raised an objection to Mozilla.

Firefox removal is completely unrelated. Also they don't specifically target OldTwitter, it's targetting all unsigned requests to Twitter API, which only happen if your browser failed to get their generator script for some reason (main causes are: old browser, social network tracker blockers settings in extensions and plugins to block them)

**Comment by notzetkin** [comment]
Okay, but what am i supposed to do then? Happened to me almost a week ago, finally was pointed to this issue, appeals, as it was said before, do nothing. What is there to do? Don't think creating a new account will help because my second account was banned in minutes after i switched to it.

**Comment by itsTyrion** [comment]
My account I used since 2017 also got suspended, no reason given, appeal automatically denied after < 20 sec. Could be nothing, but there's a non-zero chance it's related. I've also had "unusual activity" a few times out of the blue on PC and on PC only so maybe it triggers some idiotic useless spam check?

**Comment by defski** [comment]
> How would they even know you're using an extension... I don't they would care, otherwise they'd block the api altogether.

lol nevermind, my account just got locked, and I had to solve an annoying captcha to unlock it, then it threatened to have my account suspended if it happened again. Removing this extension, I just can't risk losing my account from 2011 altogether. Thanks for the hard work, dev, but this is farewell.

**Comment by rx65m** [comment]
My account got suspended after several months using this extension. I appealed and after a couple of weeks they gave my account back.
They didn't say why they suspended me.
I'm still using this extension.

**Comment by micro2547** [comment]
just got suspended permanently for using the extension again, what should i say to appeal this?

**Comment by infinitephantasm** [comment]
> what should i say to appeal this?

as far as i know, you can't say anything to appeal an auto-ban, at least for me. you'll just instantly get rejected every time with zero way to re-attain the account.

i disabled the extension out of safety, but @micro2547 what version were you using, and were you experiencing anything weird before you got suspended? (i.e. increased security/CAPTCHA checks or anything like that)

**Comment by micro2547** [comment]
i was using the latest version (1.8.9) as of writing this and as of when i got suspended. and no, i wasn't experiencing any security/captcha checks before i got suspended, it just randomly happened out of nowhere.

**Comment by infinitephantasm** [comment]
hrm. i'll keep it disabled on my end just in case, and hopefully there'll at least be some reports of similar issues coming in from others if there's some sort of correlation.

**Comment by Renkoto** [comment]
As far i can tell, third-party clients have been banned from Twitter in the early 2023, according to the X's [developer agreement](https://developer.x.com/en/developer-terms/agreement).

![Image](https://github.com/user-attachments/assets/a8308fa9-4d69-4c0a-84df-c391c9b7528e)

Maybe that has something to do with it, I don't know if there is any similar clause in the ToS (most likely it is) other services also bans third-party clients, such as Discord but in the case of Discord it is rarely enforced, generally Discord only bans users who spam the API (including through self-botting) rather than for using custom clients. 

This may be a similar case.

**Comment by samiscoolest** [comment]
happened again, suspended for "inauthentic behaviors"

**Comment by theburpman** [comment]
I was suspended too like an hour ago. "Inauthentic behaviors". I was like "no way, its because of OldTwitter? after all these years?" and then I searched Dim's Github...

Holy shit I'm super mad right now. Of course is not Dim's fault but I just lost my 16 years Twitter account. Is there's nothing left to do?

**Comment by DieTL95** [comment]
Same here, it's a bit of a bummer, I appealed but I dk if that will result in anything

**Comment by sysuah** [comment]
![Image](https://github.com/user-attachments/assets/1d849939-d358-4f50-bd4f-3b3bfca548ca)
I hate this site so much I just wanna follow my favourite artists and now I got hit with this😢
Appealed but got nothing so far

**Comment by micro2547** [comment]
is there any way i can appeal this?

**Comment by Renkoto** [comment]
> I would not bother with appealing, I've sent new ones irregularly and it's went from instant rejection to straight up ghosting now.

As I said, Xitter banned third-party clients since the early 2023, so most likely that's the reason you got banned, they're following the WhatsApp's path.
Elon Musk ruining everything as usual.

**Comment by levit4t3** [comment]
Reviving this thread again. My account was banned on march 26th for "inauthentic behavior" and the only thing I can tie it to is this extension. Used the Firefox browser client on Android for nearly 2 months with no issue until now. I haven't been able to appeal and all my pleas are dismissed. My account hasn't been reinstated and I lost nearly 5 years of progress because of this. I'm still pestering X's moderation team with appeals tho it's unlikely they'll respond. Slightly hopeful tho. I'm very disappointed

Please think twice before downloading this extension. Sadly it isn't the Dev's fault but X is getting stricter with their policies and evading them is getting harder by the day.
